core/src/main/java/org/apache/calcite/sql/SqlDelete.java

```diff
@@ -44,12 +46,14 @@ public class SqlDelete extends SqlCall {
   public SqlDelete(
       SqlParserPos pos,
       SqlNode targetTable,
+      @Nullable SqlNodeList usingList,
```

Comment:
Can you add a new constructor to this or utilize a setter to set this value? Just so we don't have to change the invocation of this function in other places.

Reply:
Done